src/onegov/people/cli.py

```diff
@@ -167,3 +167,32 @@ def _import(request: CoreRequest, app: Framework) -> None:
         click.secho(f'Imported {count} person(s)', fg='green')
 
     return _import
+
+
+@cli.command('list')
+def list_people() -> Callable[[CoreRequest | Framework], None]:
+
+    def _list(request: CoreRequest, app: Framework) -> None:
+        session = app.session()
+        properties = [
+            ('Funktion', 'function'),
+            ('Standortadresse', 'location_address', 'location_code_city'),
+            ('Postadresse', 'postal_address', 'postal_code_city'),
+            ('E-Mail', 'email'),
+            ('Telefon', 'phone'),
+            ('Telefon direkt', 'phone_direct'),
+            ('Telefon intern', 'phone_internal'),
+        ]
+        for p in session.query(Person):
+            click.secho(f'{p.title}', fg='green')
+            for label, *prop in properties:
+                value = ', '.join(
+                    str(getattr(p, attr))
+                    for attr in prop
+                    if hasattr(p, attr) and getattr(p, attr) != '')
```

Comment:
```suggestion
                    value
                    for attr in prop
                    if (raw_value := getattr(p, attr, None)) is not None
                    if (value := str(value))
```
I think we should check for `None`, unless you wanted ti be included as `"None"`?

Let's also reduce the number of times we try to access the same attribute via `hasattr`/`getattr`.

Reply:
Nice!